### ISSUE TITLE
Problems found during Integration

### ISSUE THREAD

**Issue by Philipp66904** [THI-Drone/qrcode_detection_package#4]
I found the following problems while integrating and testing your node in the simulator:

# Typo
"~recieved~" -> "**received**"

# Timer
Your configure your to be called every 0.01s:
```python
main_timer = self.create_timer(
            0.01, self.main)
```

This equals 100 Hz or 100 FPS that you try to decode.
This is way too fast and results into many problems, because the whole system is struggling to keep up and therefore for example MavSDK heartbeat messages time out (which results in a crash of the FCC Bridge, so this is a major problem).

To sum it up: We need to lower the rate.
I tried 0.1s and it worked fine and as our current position etc. is also sent at a rate of 10 Hz by the FCC bridge, I think this value is reasonable.

# Debug Output
Sending the current state at the current rate is, as we already discussed on Tuesday, not necessary.
I think, you can simply remove those outputs as you're outputting your state changes anyway, which is completely enough.

**Comment by Philipp66904** [comment]
BTW: After fixing these things, it worked flawlessly in the simulation.

**Comment by finn-ehlers** [comment]
Thanks for the feedback. I didn't think about the interval to long. 10Hz sounded a bit low to me, because we would stay a bit behind the technical possibility but you are right, 100Hz is probably way to high. I think we discuss trying 20Hz, if it doesn't work, we can fall back to 10Hz. 
I will remove the unneccessary info debug aber fix the typo ;)

**Comment by Philipp66904** [comment]
Yes, we can try 20 Hz and see how the Pi handles it.

**Comment by finn-ehlers** [comment]
okay, changes are pushed. I am planning to do a small PR today anyway, so that a working version is on dev
